client/blocks/login/continue-as-user.jsx

```diff
@@ -50,23 +51,37 @@ function ContinueAsUser( { currentUser, redirectUrlFromQuery } ) {
 
 	const userName = currentUser.display_name || currentUser.username;
 
+	const handleChangeAccount = e => {
+		e.preventDefault();
+		onChangeAccount();
+	};
+
 	// Render ContinueAsUser straight away, even before validation.
 	// This helps avoid jarring layout shifts. It's not ideal that the link URL changes transparently
 	// like that, but it is better than the alternative, and in practice it should happen quicker than
 	// the user can notice.
-	const redirectLink = (
-		<a href={ validatedRedirectUrl || '/' }>
-			<Gravatar user={ currentUser } size={ 16 } />
-			{ userName }
-		</a>
-	);
 
 	return (
 		<div className="continue-as-user">
-			{ translate( 'or continue as {{userName/}}', {
-				components: { userName: redirectLink },
-				comment: 'Alternative link under login header, skips login to continue as current user.',
-			} ) }
+			<Gravatar user={ currentUser } imgSize={ 200 } size={ 200 } />
```

Comment:
Can we use imgSize `400` with `size` 200 to ensure this renders nicely on retina?